docs/operations/live_migration.md

```diff
@@ -151,7 +151,94 @@ These values can be change in the `kubevirt` CR:
           disableTLS: false
 ```
 
-# Migration timeouts
+## Using a different network for migrations
+
+Live migrations can be configured to happen on a different network than
+the one Kubernetes is configured to use.
+That potentially allows for more determinism, control and/or bandwidth,
+depending on use-cases.
+
+### Creating a migration network on a cluster
+
+A separate physical network is required, meaning that every node on the
+cluster has to have at least 2 NICs, and the NICs that will be used for
+migrations need to be interconnected, i.e. all plugged to the same switch.
+The examples below assume that `eth1` will be used for migrations.
+
+It is also required for the Kubernetes cluster to have
+[multus](https://github.com/k8snetworkplumbingwg/multus-cni.git) installed.
+
+If the desired network doesn't include a DHCP server, then
+[whereabouts](https://github.com/k8snetworkplumbingwg/whereabouts) will
+be needed as well.
+
+Finally, a NetworkAttachmentDefinition needs to be created in the
+namespace where KubeVirt is installed. Here is an example:
+```
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: migration-network
+  namespace: kubevirt
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "migration-bridge",
+      "type": "macvlan",
+      "master": "eth1",
+      "mode": "bridge",
+      "ipam": {
+        "type": "whereabouts",
+        "range": "10.1.1.0/24"
+      }
+    }'
+```
+
+### Configuring KubeVirt to migrate VMIs over that network
+
+This is just a matter of adding the name of the
+NetworkAttachmentDefinition to the KubeVirt CR, like so:
+```
+    apiVersion: kubevirt.io/v1
+    kind: Kubevirt
+    metadata:
+      name: kubevirt
+      namespace: kubevirt
+    spec:
+      configuration:
+        developerConfiguration:
+          featureGates:
+          - LiveMigration
+        migrations:
+          network: migration-network
+```
+
+That change will trigger a restart of the virt-handler pods, as they
+get connected to that new network.
+
+From now on, migrations will happen over that network.
+
+### Configuring KubeVirtCI for testing migration networks
+
+Developers and people wanting to test the feature before deploying
+it on a real cluster might want to configure a dedicated migration
+network in KubeVirtCI.
+
+KubeVirtCI can simply be configured to include a virtual secondary
+network, as well as automatically install multus and whereabouts.
+The following environment variables just have to be declared before
+running `make cluster-up`:
+```
+export KUBEVIRT_NUM_NODES=2;
+export KUBEVIRT_NUM_SECONDARY_NICS=1;
+export KUBEVIRT_DEPLOY_ISTIO=true;
+export KUBEVIRT_WITH_CNAO=true
+```
+
+Here is a concrete example:
+https://asciinema.org/a/4Xszo18C1VzL8yBkyUaJfYSM7
```

Comment:
Unfortunately, this does not get converted to a clickable link by netlify. Could you mark it off in [] () instead?

Reply:
Doh, the demo got archived... So I just removed the link, thanks for the review and sorry for the delay!